java-client/src/main/java/org/opensearch/client/opensearch/core/TermvectorsResponse.java

```diff
@@ -157,8 +157,10 @@ protected void serializeInternal(JsonGenerator generator, JsonpMapper mapper) {
 		generator.writeKey("took");
 		generator.write(this.took);
 
-		generator.writeKey("_version");
-		generator.write(this.version);
+        if (null != this.version) {
```

Comment:
Tabs vs. spaces looks suspicious here. Also surprised spotless didn't catch it, mind checking please?